src/applications/claims-status/tests/components/ClaimStatusPage.unit.spec.jsx

```diff
@@ -183,128 +208,42 @@ describe('<ClaimStatusPage>', () => {
           });
         },
       );
-
-      context(
```

Comment:
I'm assuming we would have written tests for these using Lighthouse if that was relevant, so I'm removing these